frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -279,7 +286,7 @@ const ReceiverTableRow: React.FC<ReceiverTableRowProps> = ({
     <TableRow id={index} index={index} trKey={key} style={style}>
       <TableData className={tableColumnClasses[0]}>{receiver.name}</TableData>
       <TableData className={tableColumnClasses[1]}>
-        {!integrationTypesLabel ? (
+        {(receiver.name === 'Critical' || receiver.name === 'Default') && !integrationTypesLabel ? (
```

Comment:
I see `.name === 'Critical'` several times across these files.  Do you want to extract into a `isCritical(thingWithName)` func, and then make the `Critical` and `Default` an enum?

Reply:
Added InitialReceivers enum.  Not sure `isCritical()` is needed as the conditional seems easier to read to me, is only used in two places, and matches how we test for Default initial receiver.